misc/selinux/cfengine-enterprise.te

```diff
@@ -614,6 +614,52 @@ allow cfengine_httpd_t shell_exec_t:file map;
 allow cfengine_httpd_t shell_exec_t:file { execute execute_no_trans };
 
 
+#============= cfengine_apachectl_t ==============
```

Comment:
Only because i still feel like a SELinux newb, i kind of want a comment for each one of the allow lines explaining what it does in plain english but that seems like overkill and I can kind of follow.

Reply:
Yeah... Reasons behind some of those lines are quite unclear and TBH, I didn't spend hours figuring out why we need every single one of them. Things like `sssd_t` come from NSS plugins and there are many other actions happening behind the scenes.